submissions/ustymchyk/friends-app/js/api.js

```diff
@@ -0,0 +1,19 @@
+export class Api {
+  aliases = { list: "https://pokeapi.co/api/v2/pokemon?limit=50" };
+
+  constructor() { }
+
+  async getPokemons() {
+    const response = await fetch(this.aliases.list);
```

Comment:
I'd recommend you adding `try-catch` for checking cases if it will be failed to get pokemons list